src/main/scala/it/pagopa/interop/authorizationmanagement/api/impl/KeyApiServiceImpl.scala

```diff
@@ -43,8 +43,8 @@ final case class KeyApiServiceImpl(
   private implicit val settings: ClusterShardingSettings = shardingSettings(entity, system)
   private implicit val implicitSharding: ClusterSharding = sharding
 
-  override def createKeys(clientId: String, key: Seq[KeySeed])(implicit
-    toEntityMarshallerKeysResponse: ToEntityMarshaller[KeysResponse],
+  override def createKeys(clientId: String, key: Seq[Key])(implicit
+    toEntityMarshallerKeysResponse: ToEntityMarshaller[Seq[Key]],
```

Comment:
I suggest to keep the current approach, returning an object that contains the sequence, instead of the sequence itself